src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -858,3 +858,66 @@ g.test('buffer,resource_offset')
       });
     }, !isValid);
   });
+
+g.test('buffer,resource_binding_size')
+  .desc(
+    `
+    Test that the buffer binding size of the BindGroup entry is equal to or less than limits.
+    'maxUniformBufferBindingSize|maxStorageBufferBindingSize' if the BindGroup entry defines
+    buffer and the buffer type is 'uniform|storage|read-only-storage'.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('type', kBufferBindingTypes)
+      .beginSubcases()
+      .expand('bindingSize', ({ type }) =>
+        type === 'uniform'
+          ? [
+              1,
+              kLimitInfo.maxUniformBufferBindingSize.default,
+              kLimitInfo.maxUniformBufferBindingSize.default +
+                kLimitInfo.minUniformBufferOffsetAlignment.default,
+            ]
+          : [
+              1,
+              kLimitInfo.maxStorageBufferBindingSize.default,
+              kLimitInfo.maxStorageBufferBindingSize.default +
+                kLimitInfo.minStorageBufferOffsetAlignment.default,
+            ]
+      )
```

Comment:
This seems to work, in local testing.
```suggestion
      // Test a size of 1 to ensure there's no alignment requirement,
      // then values just within and just above the limit.
      .expand('bindingSize', ({ type }) =>
        type === 'uniform'
          ? [
              1,
              kLimitInfo.maxUniformBufferBindingSize.default,
              kLimitInfo.maxUniformBufferBindingSize.default + 1,
            ]
          : [
              1,
              kLimitInfo.maxStorageBufferBindingSize.default,
              kLimitInfo.maxStorageBufferBindingSize.default + 1,
            ]
      )
```